pkgs/by-name/my/myanon/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, flex
+, bison
+}:
+
+stdenv.mkDerivation rec {
+  src = fetchFromGitHub {
+    owner = "ppomes";
+    repo = "myanon";
+    rev = "4d67c0a3da59dc5b8ccf77f82405fd370e0d1cc5";
+    hash = "sha256-A4Qg8RvpkKgXoGTidshUyfX6FnhPR2ih1Nyne8Y87HA=";
+  };
+  pname = "myanon";
+  version = "0.5";
+
+  nativeBuildInputs = [ autoconf automake flex bison ];
+
+  configurePhase = ''
+    ./autogen.sh
+    ./configure --prefix=$out
+  '';
+  buildPhase = ''
+    make
+  '';
+  installPhase = ''
+    make install
+  '';
+
+  meta = with lib; {
+    description = "Myanon is a mysqldump anonymizer, reading a dump from stdin, and producing on the fly an anonymized version to stdout.";
+    mainProgram = "myanon";
```

Comment:
Just here? Or am I missing something?

Reply:
Oh sorry you're right on this point :) please ignore my remark about it!